webapp/platform/client/src/client4.ts

```diff
@@ -4221,6 +4222,32 @@ export default class Client4 {
             {method: 'post', body: JSON.stringify(body)},
         )
     }
+
+    private validateClient4URL(url?: string) {
```

Comment:
Minor nit, could you rename this to `validateClientUrl`? The 4 followed by a capital letter looks a bit weird to me and our naming convention in the web app is to only capitalize the first letter of abbreviations